packages/core/src/internal/execution/abi.ts

```diff
@@ -862,3 +865,11 @@ function hasDynamicSize(paramType: ParamType): boolean {
     paramType.type === "string"
   );
 }
+
+/**
+ * A utility function to capture the cast from the `Abi` type to the
+ * ethers type.
+ */
+function toEthersAbi(abi: Abi): InterfaceAbi {
+  return abi as any[];
```

Comment:
The abi of abi type I assume to be a valid json api snippet, and so convertible to ethers unchanged.